testing/regress/ecl/key/soapcall_multihttpheader.xml

```diff
@@ -1,12 +1,22 @@
 <Dataset name='soapcallResult'>
```

Comment:
Note that the changes to this file are known changes to the white space differences because of the change to standardized xml building routines instead of hard coded xml building.  These changes shouldn't have any consequence.  They show up here because this test is meant to capture an echo of the formatting.  It helps make sure such changes are acknowledged.